bedrock/firefox/templates/firefox/new/batm/machine.html

```diff
@@ -0,0 +1,45 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
```

Comment:
Just copypasta. This macro is imported on all the other BATM variants but may not be needed on any of them. Everything still seems to work without it so it could be a holdover from some past variant that had an Android button.